docs/design/coreclr/botr/vectors-and-intrinsics.md

```diff
@@ -0,0 +1,189 @@
+Vectors and Hardware Intrinsics Support
+===
+---
+
+# Introduction
+The CoreCLR runtime has support for several varieties of hardware intrinsics, and various ways to compile code which uses them. This support varies by what processor the logic is running on, and the code executed depends on how the RyuJIT was invoked. This document describes the various behaviors of intrinsics in the runtime, and concludes with documentation on what implications this has for developers working on the runtime and libraries portions of the runtime. 
```

Comment:
I thought I'd read somewhere (not sure where) that we wanted to keep markdown line lengths below some reasonable length, eg 80 or 100?

There is decent tooling for this -- for instance VSCode markdown formatting extensions.

```suggestion
The CoreCLR runtime has support for several varieties of hardware intrinsics, and various ways to compile code which uses them. This support varies by target processor, and the code produced depends on how the jit compiler is invoked. This document describes the various behaviors of intrinsics in the runtime, and concludes with implications  for developers working on the runtime and libraries portions of the runtime. 
```